portal/models/user.py

```diff
@@ -692,6 +688,21 @@ def locale_name_from_code(locale_code):
 
         return locale_options
 
+    @property
+    def lockout_period_minutes(self):
```

Comment:
We usually just reference the `config` dict directly. I think I like this approach better though, makes things a little more decoupled

Reply:
Same here. I was referencing it in multiple places at first and it felt like I was relying too much on the underlying implementation.